### ISSUE TITLE
Request body assumed always ref

### ISSUE THREAD

**Issue by synclpz** [koxudaxi/fastapi-code-generator#27]
Seems like codegen (0.0.13) expects:

```yaml
      requestBody:
        content:
          application/json:
            schema:
              $ref: "#/components/schemas/Something"
```

and does not accept:

```yaml
      requestBody:
        content:
          application/json:
            schema:
              type: string
```

Log:

```
Traceback (most recent call last):
  File "/Users/victor/megad-heated-floor-thermostat/.venv/bin/fastapi-codegen", line 8, in <module>
    sys.exit(app())
  File "/Users/victor/megad-heated-floor-thermostat/.venv/lib/python3.8/site-packages/typer/main.py", line 213, in __call__
    return get_command(self)()
  File "/Users/victor/megad-heated-floor-thermostat/.venv/lib/python3.8/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/Users/victor/megad-heated-floor-thermostat/.venv/lib/python3.8/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/Users/victor/megad-heated-floor-thermostat/.venv/lib/python3.8/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/victor/megad-heated-floor-thermostat/.venv/lib/python3.8/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/Users/victor/megad-heated-floor-thermostat/.venv/lib/python3.8/site-packages/typer/main.py", line 496, in wrapper
    return callback(**use_params)  # type: ignore
  File "/Users/victor/megad-heated-floor-thermostat/.venv/lib/python3.8/site-packages/fastapi_code_generator/__main__.py", line 26, in main
    return generate_code(input_name, input_text, output_dir, template_dir)
  File "/Users/victor/megad-heated-floor-thermostat/.venv/lib/python3.8/site-packages/fastapi_code_generator/__main__.py", line 37, in generate_code
    parsed_object: ParsedObject = parser.parse()
  File "/Users/victor/megad-heated-floor-thermostat/.venv/lib/python3.8/site-packages/fastapi_code_generator/parser.py", line 391, in parse
    return self.parse_paths(openapi)
  File "/Users/victor/megad-heated-floor-thermostat/.venv/lib/python3.8/site-packages/fastapi_code_generator/parser.py", line 400, in parse_paths
    return ParsedObject(
  File "/Users/victor/megad-heated-floor-thermostat/.venv/lib/python3.8/site-packages/fastapi_code_generator/parser.py", line 372, in __init__
    operation.arguments
  File "/usr/local/opt/python@3.8/Frameworks/Python.framework/Versions/3.8/lib/python3.8/functools.py", line 967, in __get__
    val = self.func(instance)
  File "/Users/victor/megad-heated-floor-thermostat/.venv/lib/python3.8/site-packages/fastapi_code_generator/parser.py", line 175, in arguments
    return self.get_arguments(snake_case=False)
  File "/Users/victor/megad-heated-floor-thermostat/.venv/lib/python3.8/site-packages/fastapi_code_generator/parser.py", line 183, in get_arguments
    argument.argument for argument in self.get_argument_list(snake_case)
  File "/Users/victor/megad-heated-floor-thermostat/.venv/lib/python3.8/site-packages/fastapi_code_generator/parser.py", line 197, in get_argument_list
    if self.request:
  File "/usr/local/opt/python@3.8/Frameworks/Python.framework/Versions/3.8/lib/python3.8/functools.py", line 967, in __get__
    val = self.func(instance)
  File "/Users/victor/megad-heated-floor-thermostat/.venv/lib/python3.8/site-packages/fastapi_code_generator/parser.py", line 113, in request
    type_hint=schema.ref_object_name,
  File "/Users/victor/megad-heated-floor-thermostat/.venv/lib/python3.8/site-packages/datamodel_code_generator/parser/jsonschema.py", line 115, in ref_object_name
    return self.ref.rsplit('/', 1)[-1]  # type: ignore
AttributeError: 'NoneType' object has no attribute 'rsplit'
```

**Comment by koxudaxi** [comment]
@synclpz 
Thank you for creating this issue.
I will fix it this week.

**Comment by koxudaxi** [comment]
I'm sorry. I don't forget the issue.
I will try it after I resolve the blocker. https://github.com/koxudaxi/fastapi-code-generator/issues/35

**Comment by Baukebrenninkmeijer** [comment]
Also running into this issue. Any update on this @koxudaxi?

**Comment by koxudaxi** [comment]
@Baukebrenninkmeijer 
Thank you for posting the comment.
I will work on it this weekend.

**Comment by koxudaxi** [comment]
@synclpz 
I have released a fixed version as `0.0.14`
